install.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
 set -u
 
-# This script installs to /usr/local only. To install elsewhere (which is
-# unsupported) you can untar https://github.com/Homebrew/brew/tarball/master
+# First check if the OS is Linux.
+if [[ "$(uname)" = "Linux" ]]; then
+    HOMEBREW_ON_LINUX=true
+else
+    HOMEBREW_ON_LINUX=false
```

Comment:
Would strongly suggest not setting this on macOS so you can do `[ -z "$HOMEBREW_ON_LINUX" ]`. Alternatively, might be a bit more readable to do `MACOS` (and `LINUX`) or `HOMEBREW_ON_MACOS`.